src/course-search/styles/_FacetList.scss

```diff
@@ -40,4 +40,35 @@
       }
     }
   }
+
+  .typeahead {
+    .dropdown-menu {
+      .facet-list & {
+        padding: 50px 0 0;
+        max-height: none;
+        overflow: visible;
+        width: 420px;
+      }
+
+      .typeahead-dropdown-menu-scrollable-items {
+        position: relative;
+        max-height: 200px;
+        overflow: auto;
+
+        .no-options-found {
+          display: block;
+        }
+      }
+
+      .typeahead-dropdown-input {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 98%;
```

Comment:
[curious] Why does the input only take up 98% of the width?

Reply:
If we use `width: 100%` than focused border goes outside the area of menu. Please see the search input box inside menu in below screenshots. @adamstankiewicz can you suggest a good solution for this?

### Width set to `100%`
<img width="871" alt="Screenshot 2021-03-22 at 11 25 59 PM" src="https://user-images.githubusercontent.com/6767924/112039608-09aa1a80-8b66-11eb-9c7b-d619817318ba.png">

---

### Width set to `98%`
<img width="597" alt="Screenshot 2021-03-22 at 11 27 13 PM" src="https://user-images.githubusercontent.com/6767924/112039770-39592280-8b66-11eb-90b4-7952e7ed6ae9.png">

Reply:
@muhammad-ammar A few things here...

```scss
.dropdown-menu {
  .facet-list & {
    padding: 50px 0 0;
    max-height: none;
    overflow: visible;
    width: 420px;
  }
}
```

I don't believe these above styles (^^) are actually doing anything as the ordering of these nested classes are actually reversed in the DOM (i.e., `.dropdown-menu` is a child of `.facet-list` however these styles are written to expect the reverse and won't apply as a result).

You shouldn't need to modify the positioning of the the "Find a skill..." input to be absolute or even specify a width for it as it should naturally fill up all available width. The scrollable items can just naturally sit underneath the default positioning of the input.

Modifying your styles locally, I landed on the following:

```scss
.typeahead {
  .dropdown-menu {
    max-height: unset;

    .typeahead-dropdown-menu-scrollable-items {
      position: relative;
      max-height: 250px;
      overflow-y: scroll;
    }
  }
}
```

<img src="https://user-images.githubusercontent.com/2828721/112055859-bc11bb80-8b2d-11eb-8a3f-91beab92ad7a.png" width="500" alt="skills facet dropdown styles" />

Reply:
@adamstankiewicz The above styles are not working on my side. Please see the screenshot

<img width="854" alt="Screenshot 2021-03-23 at 2 15 43 AM" src="https://user-images.githubusercontent.com/6767924/112059481-b47a0300-8b7d-11eb-90c9-0ff1a786a052.png">

I am not sure how to proceed on this. This is getting blocked for me.

Reply:
@adamstankiewicz I have checked with https://www.sassmeister.com/ and 

```scss
.dropdown-menu {
  .facet-list & {
    padding: 50px 0 0;
    max-height: none;
    overflow: visible;
    width: 420px;
  }
}

```

will be converted to

```css
.facet-list .dropdown-menu {
  padding: 50px 0 0;
  max-height: none;
  overflow: visible;
  width: 420px;
}
```

Reply:
@muhammad-ammar, I drafted this PR (could be merged into this PR) with more concrete style suggestions to simplify things a bit here after pulling your branch down locally and testing with the local checkout of frontend-enterprise: https://github.com/edx/frontend-enterprise/pull/80

It is primarily changes in _FacetList.scss and TypeaheadFacetDropdown.jsx though there's a couple other clean up things that could happen after upgrading Paragon to latest. Largely, it removes the unneeded absolute positioning on the typeahead `input` element.

Reply:
@adamstankiewicz Thank You! Can you share how you test/view style changes in `frontend-enterprise`?

Reply:
@muhammad-ammar If you pull the latest from `master` in frontend-app-learner-portal-enterprise, you'll get some style updates that @raq929 merged today to import the styles from the `frontend-enterprise` package (take a look at `src/index.scss` to see the styles imported).

You'll also need to `npm i -s lodash.debounce` into frontend-app-learner-portal-enterprise so the peerDependency introduced in `frontend-enterprise` will resolve.

Then, checkout the branch from #80 in your local `frontend-enterprise` (or merge it into your branch), and use the module.config.js approach you have been doing to alias `@edx/frontend-enterprise` to your local checkout of the package.

You noted earlier that you need to run `make build` any time you make changes in `frontend-enterprise`. You could use the `src` directory instead of `dist` in module.config.js to get hot reloading working as you modify `frontend-enterprise`, e.g.:

```
{ moduleName: '@edx/frontend-enterprise', dir: '../frontend-enterprise', dist: 'src' },
```